tests/image/image.cpp

```diff
@@ -84,7 +84,35 @@ TEST(ZincFieldImage, read_png)
 	Streamresource sr = si.createStreamresourceFile(TestResources::getLocation(TestResources::IMAGE_PNG_RESOURCE));
 	EXPECT_TRUE(sr.isValid());
 
-	EXPECT_EQ(OK, im.read(si));
+	EXPECT_EQ(CMZN_OK, im.read(si));
+}
+
+TEST(ZincFieldImage, set_get_buffer)
+{
+	ZincTestSetupCpp zinc;
+
+	FieldImage im = zinc.fm.createFieldImage();
+	EXPECT_TRUE(im.isValid());
+
+	EXPECT_EQ(CMZN_OK, im.setPixelFormat(FieldImage::PIXEL_FORMAT_LUMINANCE));
```

Comment:
Test default values for PixelFormat and NumberOfBitsPerComponent before setting. Even if invalid.